tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/service/impl/ProgrammeMembershipServiceImpl.java

```diff
@@ -93,26 +112,36 @@ public ProgrammeMembershipDTO save(ProgrammeMembershipDTO programmeMembershipDTO
   /**
    * Save a list of programmeMembership.
    *
-   * @param programmeMembershipDTO the list of entities to save
+   * @param programmeMembershipDto the list of entities to save
    * @return the list of persisted entities
    */
   @Override
-  public List<ProgrammeMembershipDTO> save(List<ProgrammeMembershipDTO> programmeMembershipDTO) {
-    log.debug("Request to save ProgrammeMembership : {}", programmeMembershipDTO);
+  public List<ProgrammeMembershipDTO> save(List<ProgrammeMembershipDTO> programmeMembershipDto) {
```

Comment:
I'm thinking if we might need to add @Transactional annotation for save/delete.

Reply:
It's declared at the class level, so it would apply here as well :)